doc/generate-kuadrant-auth-policy.md

```diff
@@ -38,24 +42,20 @@ components:
   securitySchemes:
     securedDog:
       type: openIdConnect
-      openIdConnectUrl: https://example.com/.well-known/openid-configuration
+      openIdConnectUrl: https://sso.example.com/auth/realms/petstore
 ```
 
-Running the command
+Take this example and save it as `example.yaml` and than run the command:
 
-```
-kuadrantctl generate kuadrant authpolicy --oas ./petstore-openapi.yaml  | yq -P
+```bash
+kuadrantctl generate kuadrant authpolicy --oas example.yaml
```

Comment:
i dont really get this change, why change the name to be example and why use remove the pretty print?

Reply:
The `yp` command has been removed from multiple guides as is now not needed. The pretty print is done as default.
I changed the name of the yaml file so it does not clash with the same name as the yaml file generated as part of User guide that follows later. I am opened to better file name suggestion though.